packages/react-components/src/InputRpc/options/method.tsx

```diff
@@ -25,7 +25,7 @@ export default function createOptions (api: ApiPromise, sectionName: string): Dr
     })
     .map((value): DropdownOption => {
       const { description, params } = section.methods[value];
-      const inputs = params.map(({ name }): string => name).join(', ');
+      const inputs = params.map(({ name }: { name : string }): string => name).join(', ');
```

Comment:
Could you plz give an example on how the mapping function is converting inputs? @KarishmaBothara

Reply:
Haven't changed anything here..just added a type to the object { name } : { name: string }